app/public/presentation/common/courses/course.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, Input } from '@angular/core';
-import { Course } from './view-model/course.view-model';
+import { Course } from '../../../domain/entities/course.entity';
```

Comment:
We can use `baseUrl` to avoid using relative paths ([reference](http://www.typescriptlang.org/docs/handbook/compiler-options.html))